.github/pr_templates/pull_request_template.md

```diff
@@ -0,0 +1,4 @@
+TODO: PR description and edit branch pointing to
+      gsy-backend-integration-tests repository.
```

Comment:
@hannesdiedrich @spyrostz which template do you think we could use? I was thinking of something simple like:
```md
## Reason for the proposed changes

Please describe what we want to achieve and why.

## Proposed changes

-

<!--- If needed, choose which branch of the gsy-backend-integration-tests repository will be used to run integration tests. Please only edit the name of the branch. -->
**INTEGRATION_TESTS_BRANCH**=master
```
What do you think?

Reply:
@spyrostz @hannesdiedrich @fievelk 
I'd like to bring this conversation back to life whenever you have time just to finish this PR.

Reply:
I agree with the template Pierpaolo proposes.

Reply:
+1 from me too!

Reply:
Awesome, I adapted the template message on [d2bdbe0](https://github.com/gridsingularity/gsy-e/pull/1394/commits/d2bdbe0c4f6f76c4bdbb7a43b523a22d8cb97bb8).